examples/1_type_test.toml

```diff
@@ -1,61 +1,64 @@
-[litmus]
-
-# this test checks if a 200 content response is obtainable from the server
-[[litmus.test]]
-name="httpbin get - check code"
-method="GET"
-url="https://{{.base_service_url}}/get"
-wants_code=200
-
-# this test checks if the body contains the "Connection" field, set to the value "close"
-[[litmus.test]]
-name="httpbin get - check body"
-method= "GET"
-url= "http://{{.base_service_url}}/get"
-[litmus.test.query]
-foo = "bar"
-baz = "qux"
-[[litmus.test.getters]]
-type="body"
-path="headers.Connection"
-exp="close"
-[[litmus.test.getters]]
-type="body"
-path="headers.Connection"
-exp="close"
-set="some_key"
-
-# etc. etc. commenting tests is a good idea.
-[[litmus.test]]
-name= "httpbin post - returns post data"
-method= "POST"
-url="http://{{.base_service_url}}/post"
-wants_code= 200
-body='''
-{"from_env":"{{.example_value}}","test":"{{.some_key}}"}
-'''
-[litmus.test.headers]
-Content-Type = "application/json"
-[[litmus.test.getters]]
-type="body"
-path="json.from_env"
-exp="some example value"
-[[litmus.test.getters]]
-type="body"
-path="json.test"
-exp="close"
-
-[[litmus.test]]
-name="httpbin - check gzip headers"
-method= "GET"
-url="https://{{.base_service_url}}/gzip"
-wants_code=200
-[[litmus.test.getters]]
-type="body"
-path="gzipped"
-exp="true"
-[[litmus.test.getters]]
-type="head"
-path="Content-Type"
-exp="application/json"
-
+#[litmus]
```

Comment:
Same as commenting code above.

Reply:
this whole file needs a complete re-write